metabase/metabase
# FE: Show a checkbox near parameters in Action Editor Form

Epic: https://github.com/metabase/metabase/issues/30016

### Description

Adds ability to hide action fields from ActionCreator form for both implicit and query actions

### How to verify

- Go to model -> actions -> edit action or `new +` -> create action 
- uncheck "show field", save
- open action again - show field should be unchecked

### Demo

https://www.loom.com/share/8f445ffb8e7e4eb2b631f21dfd991140

### Checklist
